app/assets/stylesheets/home.scss

```diff
@@ -18,41 +19,41 @@
    -moz-box-shadow: 0 0 10px #B3B3B3;
    -webkit-box-shadow: 0 0 10px #B3B3B3;
    box-shadow: 0 0 10px #B3B3B3;
+   z-index: 100;
 
 	h2 {
 		color: #85BEAA;
 	}
 }
 
-.new {
-	margin-top: 20px;
-	margin-left: 20px;
-	cursor: pointer;
-	border: 3px solid #85BEAA;
-	background-color: #85BEAA;
-	padding: 10px 20px 10px 20px;
-	transition: all 0.5s;
-	text-align: center;
+/* NEW BUTTON */
 
-	span{
-		color: #E6E6E6;
-		font-size: 16px;
-		transition: all 0.5s;
-	}
+.new {
+  cursor: pointer;
+  background-color: #85BEAA;
```

Comment:
You should probably make the colors constants like "teal" or "blue-green" or something (since you're using the same color in more than one place).